cloud/ec2.go

```diff
@@ -113,13 +113,15 @@ func (s *EC2ProviderSettings) fromDistroSettings(d distro.Distro) error {
 	return nil
 }
 
-func (s *EC2ProviderSettings) getSecurityGroups() []*string {
+func (s *EC2ProviderSettings) getSecurityGroups(settings *evergreen.Settings) []*string {
 	groups := []*string{}
 	if len(s.SecurityGroupIDs) > 0 {
 		for _, group := range s.SecurityGroupIDs {
 			groups = append(groups, aws.String(group))
 		}
-		return groups
+	} else {
```

Comment:
You can still return groups in the if statement, and then drop the else and return the default if you get that far.

Reply:
Fixed!